std/algebra/fields_bls24315/e24_test.go

```diff
@@ -405,7 +406,7 @@ func TestFrobeniusFp24(t *testing.T) {
 }
 
 // benches
-var ccsBench frontend.CompiledConstraintSystem
+var ccsBench constraint.ConstraintSystem
```

Comment:
Unrelated to this PR, but shouldnt ccsBench vars be in function scope instead of being global? There is a race condition when benchmarks are run in parallel?

Reply:
why would you run benchmarks in parallel? ^^
Doing this limits compiler optimizations in some cases; see last paragraph here --> https://dave.cheney.net/2013/06/30/how-to-write-benchmarks-in-go